scripts/kconfig/kconfiglib.py

```diff
@@ -2928,7 +2928,7 @@ def _parse_block(self, end_token, parent, prev):
                 node = MenuNode()
                 node.kconfig = self
                 node.item = sym
-                node.is_menuconfig = (t0 is _T_MENUCONFIG)
```

Comment:
Or should we leave this file as-is instead and merge this with superpower? cc @nashif

Reply:
after you reverted the changes pointed out in this comment https://github.com/zephyrproject-rtos/zephyr/pull/63973#discussion_r1368619501

then it should be as simple as dropping this commit https://github.com/zephyrproject-rtos/zephyr/pull/63973/commits/6ca708fb1d735482a73212de6c1f819a0145ba66 because you are no longer modifying kconfiglib.py.
So therefore CI should still be green (not considering other potential failures) 😉